### ISSUE TITLE
Noms

### ISSUE THREAD

**Issue by BrettRey** [nert-nlp/cgel#50]
From John Payne: "The one way that CGELBank deviates quite significantly from CGEL itself , it strikes me, is in the treatment of the phrasal category Nom. Quite specifically, Rodney and I did not want CGEL Nom to be identified with the X-bar construct N-bar. Thus in CGEL, it is not the case that Complements are invariably sisters of N and Modifiers are invariably sisters of Nom. If there is just one Modifier, or a Modifier is the lowest dependent, it will be sister of N and daughter of Nom. And if a Complement is not the lowest dependent it will be a sister of Nom. This allows for relatively free ordering of Complements and Modifiers of similar weight (e.g. his exit in haste from the bar) without the necessity for gaps and postposing. It is also compatible with the idea that the distinction between Complements and Modifiers in NP structure is far from secure (no supposed Complements are obligatory; anaphoric one is not an N-bar replacement), as argued explicitly in ‘Anaphoric one and its implications’ (John Payne, Geoffrey K. Pullum, Barbara C. Scholz and Eva Berlage). Language 89.4, 2013, 794-829. Note then that your tree (40) does not reflect CGEL (see CGEL page 1061 for a corresponding CGEL tree). Going back to the N-bar concept strikes me as a retrograde step."

John was the primary author on chapter 5: Nouns and noun phrases. Geoff prefers the N-bar concept but was not an author on chapter 5.

**Comment by nschneid** [comment]
Hmm. I can see the logic of the N-bar concept from a theoretical perspective, and if the goal is to keep the number of grammar rules to a bare minimum. From a practical descriptive perspective, it is kind of tedious to have all those extra unary levels. People might find the data easier to understand without them. Also, the current approach creates puzzles like medial Mods in #49.

One option is to revise the Lexical Projection Principle to say that every lexeme (besides Coordinators/Sdrs) projects a phrasal category, which may contain complements OR a modifier (and modifiers always adhere to binary branching). Would that be straightforward to apply or would it open up new questions?

**Comment by nschneid** [comment]
Looking at his comment I think he's also saying that Complements should be binary-branching within the Nom, and can interleave with Mods to avoid the need for postposing. I don't have a strong feeling about that but I assume it's different from VP structure (for ditransitives or transitive particle verbs we wouldn't want to introduce additional nesting right?).

**Comment by BrettRey** [comment]
I've started rereading the Anaphoric _one_ paper. I should have reread it some time ago. I think it will be very helpful here.

**Comment by nschneid** [comment]
It sounds like John is arguing for treating Mods and Comps the same within Noms. A revised approach could be:

1. Binary Branching Principle: CGELBank branching is no more than binary except in limited circumstances:*
    * Coordinations with more than two coordinates
    * Flat structures
    * Internal complements in VPs (Comp, Obj, Particle, etc.) should be sister to the head V
    * Supplements never add an additional layer and do not count toward the branching limit
 
   Extra layers are added (typically for Mod) where necessary in order to avoid prohibited ternary+ structures. (But a Mod may also form a binary structure with a lexical head, e.g. [AdjP [Mod:AdvP [Head:Adv very]] [Head:Adj tall]].)
2. Lexical Projection Principle: Outside of flat structures, and excepting Coordinators and Sdrs, a lexical node almost always projects a phrase of the corresponding category: i.e. N projects Nom (which *typically* projects NP), V projects VP (which *typically* projects Clause), P projects PP, Adv projects AdvP, etc. The one exception is that subject-auxiliary inversion targets auxiliaries specifically (rather than the VP they would project in normal position), so if the V_aux in Prenucleus position is monolexemic (not coordinated or modified), it will not project a VP there.


\* This assumes that AdjP, AdvP, PP, DP, IntP are more like Noms than VPs—multiple complements (if that ever happens) would be layered.

**Comment by nschneid** [comment]
@BrettRey what do you think of the above, with the special exception for auxiliaries?

**Comment by BrettRey** [comment]
Yes, I think this is clear, simple, and accurate. I also think multiple complements are possible: _a bet _[_with Nathan_] [_for $50_] [_that multiple complements are possible_]

**Comment by nschneid** [comment]
> Yes, I think this is clear, simple, and accurate. 

OK, will work on updating the data

> I also think multiple complements are possible: _a bet_ [_with Nathan_] [_for $50_] [_that multiple complements are possible_]

Sorry I meant multiple complements in an AdjP, AdvP, PP, DP, or IntP

**Comment by BrettRey** [comment]
_happy_ [_for us_] [_that we've got this far_]

**Comment by nschneid** [comment]
OK—would you want those to be layered like in a Nom, or all on the same level like in a VP?

**Comment by BrettRey** [comment]
Layered.
